package.json

```diff
@@ -6809,10 +6809,10 @@
                                             },
                                             {
                                                 "key": "m",
-                                                "name": "Run current cell",
-                                                "icon": "play",
+                                                "name": "Create R Terminal",
```

Comment:
why creating the terminal with m? Can't we assign a free letter so that we don't introduce a breaking change?

Reply:
m is not assigned in R mode only in Quarto mode. And it was me, who assigned "m" in Quarto mode. And i actually now think it makes more cell to assign Run current cell so "S" because "s" is Run selected lines. I also thought about assigned "terminal" to t but that's a default assignment for "test". Also "SPC m m" is easy to type and "Create R Terminal" is a command i have to use very often now, each time I open a script acutally.

Reply:
Wondering if this terminal is similar to `SPC m '` in https://develop.spacemacs.org/layers/+lang/ess/README.html#inferior-repl-process?

Reply:
At least from a user perspective it's comparable I think. Don't understand enough of the intricacies of VSCode and ESS to say if it really the same thing.
But since the keybindings are not really aligned to those of ESS and since I find `SPC m  '` much harder to reach than `SPC m m` i would prefer the latter.

Reply:
`SPC m m` is definitely easier to reach, but I would say `SPC m '` is more memorable since we also have `SPC '` that opens terminal.

@MarcoIeni thoughts?

Reply:
Or what about both then? `SPC m m` is not assigned yet (except in the Quarto mode, which I did and already changed). Or alternatively `SPC m i` , i for interactive + it's used in ESS in addiiton to `SPC m '` (although `SPC m m` is still my favorite)

Reply:
Also seems reasonable to me at least `SPC m m` + `SPC m '`. I am sure you favor `SPC m m` for a reason since you work with R more than I ever would probably.

Reply:
The reason is simply that it's super easy to type (so I wondered that nobody, me included, did not assign this yet) and it's a frequent command.
Haha true, unfortunately I am under the impression that there are not many VSpacecode R users (yet), I use it almost daily.